docs/cn/Running-Alluxio-on-a-Cluster.md

```diff
@@ -33,7 +33,9 @@ $ cp conf/alluxio-site.properties.template conf/alluxio-site.properties
 
 最后，同步所有信息到worker节点。你可以使用
 
-{% include Running-Alluxio-on-a-Cluster/sync-info.md %}
+```bash
+$ ./bin/alluxio copyDir <dirname>
```

Comment:
Can the file `Running-Alluxio-on-a-Cluster/sync-info.md` be removed?

Reply:
I just follow the issue's guide.  The code bellow the {% include Running-Alluxio-on-a-Cluster/sync-info.md %} is the code in the file Running-Alluxio-on-a-Cluster/sync-info.md. I just replace the %include expression with the code in sync-info.md.

Reply:
I just follow the issue's guide. The code bellow the {% include Running-Alluxio-on-a-Cluster/sync-info.md %} is the code in the file Running-Alluxio-on-a-Cluster/sync-info.md. I just replace the %include expression with the code in sync-info.md.